javascript/testing_javascript/project_battleship.md

```diff
@@ -14,17 +14,17 @@ We have not yet discussed testing the appearance of a webpage.  Doing this requi
    3. Ships should have a `hit()` function that increases the number of 'hits' in your ship.
    4. `isSunk()` should be a function that calculates it based on their length and the number of 'hits'.
 2. Create `Gameboard` factory.
-   1. Note that we have not yet created any User Interface.  We should know our code is coming together by running the tests.  You shouldn't be relying on `console.log`s or DOM methods to make sure your code is doing what you expect it to.
+   1. Note that we have not yet created any User Interface.  We should know our code is coming together by running the tests.  You shouldn't be relying on `console.log` or DOM methods to make sure your code is doing what you expect it to.
    2. Gameboards should be able to place ships at specific coordinates by calling the ship factory function.
    3. Gameboards should have a `receiveAttack` function that takes a pair of coordinates, determines whether or not the attack hit a ship and then sends the 'hit' function to the correct ship, or records the coordinates of the missed shot.
    4. Gameboards should keep track of missed attacks so they can display them properly.
    5. Gameboards should be able to report whether or not all of their ships have been sunk.
 3. Create `Player`.
    1. Players can take turns playing the game by attacking the enemy Gameboard.
-   2. The game is played against the computer, so make 'computer' players capable of making random plays.  The AI does not have to be smart, but it should know whether or not a given move is legal. (i.e. it shouldn't shoot the same coordinate twice).
+   2. The game is played against the computer, so make the 'computer' capable of making random plays.  The AI does not have to be smart, but it should know whether or not a given move is legal. (i.e. it shouldn't shoot the same coordinate twice).
 4. Create the main game loop and a module for DOM interaction.
    1. At _this_ point it is appropriate to begin crafting your User Interface.
-   2. The game loop should set up a new game by creating Players and Gameboards.  For now just populate each Gameboard with predetermined coordinates.  You can implement a system for allowing players to place their ships later.
+   2. The game loop should set up a new game by creating Players and Gameboards.  For now just populate each Gameboard with predetermined or random coordinates.  You can implement a system for allowing players to place their ships later.
```

Comment:
The other changes are fine. However, this is changing the requirements of the assignment and not quite fixing a typo. So if we could keep line 27 the same as before I would be fine with approving this. If you think the assignment would be better with random coordinates definitely set that up as an issue so the team can discuss it.

Reply:
Alright I understand that's fine with me ! Thank you for your reply !